xga/utils.py

```diff
@@ -660,10 +660,17 @@ def check_telescope_choices(telescope: Union[str, List[str]]) -> List[str]:
 ESASS_VERSION = None
 # This checks for an installation of eSASS
 eSASS_AVAIL = False
+which_evtool = shutil.which("evtool")
 if shutil.which("evtool") is None:
     warn("No eSASS installation detected on system, as such all functions in xga.generate.esass will not work.")
```

Comment:
Another warning I wrote wrong - please add the 'stacklevel=2' argument after the warning string (i.e. ....work.", stacklevel=2) - it makes them a little easier to read when the warning is triggered (my bad).

Reply:
Again I don't think that you pushed these changes if you made them